codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/OperationEndpointResolverMiddleware.kt

```diff
@@ -6,26 +6,35 @@
 package software.amazon.smithy.aws.swift.codegen.middleware
```

Comment:
Changes in this file add support for handling the `@operationContextParams` trait